pkgs/golang.org/x/tools/cmd/stringer/registry.yaml

```diff
@@ -0,0 +1,12 @@
+packages:
+  - type: go_install
+    name: golang/tools/stringer
+    aliases: # Set aliases to keep the compatibility
+      - name: golang.org/x/tools/cmd/stringer
```

Comment:
```suggestion
```

`aliases` is used only to rename packages keeping the compatibility.
This tool is newly added, so we don't have to use `aliases`.